commandbased/gearsbot/subsystems/drivetrain.py

```diff
@@ -47,22 +48,6 @@ def __init__(self, robot):
         self.rangefinder = wpilib.AnalogInput(6)
         self.gyro = wpilib.AnalogGyro(1)
 
-        wpilib.LiveWindow.addActuator(
-            "Drive Train", "Front_Left Motor", self.front_left_motor
-        )
-        wpilib.LiveWindow.addActuator(
-            "Drive Train", "Back Left Motor", self.back_left_motor
-        )
-        wpilib.LiveWindow.addActuator(
-            "Drive Train", "Front Right Motor", self.front_right_motor
-        )
-        wpilib.LiveWindow.addActuator(
-            "Drive Train", "Back Right Motor", self.back_right_motor
-        )
-        wpilib.LiveWindow.addSensor("Drive Train", "Left Encoder", self.left_encoder)
-        wpilib.LiveWindow.addSensor("Drive Train", "Right Encoder", self.right_encoder)
-        wpilib.LiveWindow.addSensor("Drive Train", "Rangefinder", self.rangefinder)
-        wpilib.LiveWindow.addSensor("Drive Train", "Gyro", self.gyro)
```

Comment:
Wow, I see nobody's maintained the gearsbot example in a long while... we'll need to actually update it against the one in allwpilib. I presume these were changed to `addChild` calls or whatever.